lib/nestcam.js

```diff
@@ -22,14 +22,14 @@ class NestCam {
 
   // Homebridge
 
-  configureWithHAP(hap, useOMX) {
+  configureWithHAP(hap, config) {
     uuid = hap.uuid;
     Service = hap.Service;
     Characteristic = hap.Characteristic;
     StreamController = hap.StreamController;
 
     let self = this;
-    self.useOMX = useOMX;
+    self.ffmpegCodec = config.ffmpegCodec;
```

Comment:
if you want to do this, let's have a fallback to default `libx264` (or omx if legacy useOMX flag is true)

Reply:
I did in one place but missed it here. Will update.

This replaces the useOMX flag entirely, so I'll add a check if it's set in the config and set the codec appropriately.